src/test/bin/winkernel/control.cpp

```diff
@@ -517,7 +517,8 @@ typedef union {
     QUIC_RUN_VN_TP_ODD_SIZE_PARAMS OddSizeVnTpParams;
     UINT8 TestServerVNTP;
     BOOLEAN Bidirectional;
-
+    BOOLEAN ServerSupport;
+    BOOLEAN ClientSupport;
```

Comment:
You need to put this in a struct. You're putting this in a `union` which means they all share the same memory. So these are always the same value.

Reply:
Done. will push